.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+
+on:
+  push:
+    branches:
+      - [main]
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.CI_GITHUB_TOKEN }}
+
+      - uses: actions/setup-node@v3
+        with:
+          always-auth: true
+          registry-url: https://registry.npmjs.org
+
+      - name: Setup Git
+        run: |
+          git config --local user.name "GitHub Actions"
+          git config --local user.email "actions@github.com"
+
+      - run: npm ci
+      - run: npm test
+
+      - run: npm run release
+        env:
```

Comment:
@thepassle, hi! If you can please set up these tokens as Secrets for this repo, we can have automatic releases for the `cookie` package set up and running.

Reply:
I've left more detailed instructions on how to create those tokens in the description to this pull request.